helix-term/src/commands.rs

```diff
@@ -1188,14 +1213,14 @@ fn search(cx: &mut Context) {
             if event != PromptEvent::Update {
                 return;
             }
-            search_impl(doc, view, &contents, &regex, false);
+            search_impl(doc, view, &contents, &regex, false, reverse);
         },
     );
 
     cx.push_layer(Box::new(prompt));
 }
 
-fn search_next_impl(cx: &mut Context, extend: bool) {
+fn search_next_or_prev_impl(cx: &mut Context, extend: bool, reverse: bool) {
```

Comment:
Just call this `search_impl`

Reply:
There's already a different fn named `search_impl`, called by this.